src/actHTMLCSS/actHTMLCSS.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+# Creates necessary directories
+def createDirs(dirs):
+    if not os.path.isdir("css"):
+            os.makedirs("css")
+    for d in dirs:
+        if not os.path.isdir("css/" + d):
+            os.makedirs("css/" + d)
+
+
+# Creates css-file file_name.css from list of (r,g,b) tuples for volume v, layer l, module m
+# Note: css-files are opened with "a" so that the file can be written to out of order
+# This means that if a css-file is not deleted before running the program again with the same data
+# it will simply add the css data to the existing file a second time
+def createCSS(file_name, colors, v, l):
+    with open(file_name + ".css", "a") as f:
+        for i in colors:
+            start = "#module_vol" + str(v) + "_lay" + str(l) + "_sen" + str(i) + "{\n"
+            fill = "    fill: rgb(" + str(colors[i][0]) + ", " + str(colors[i][1]) + ", " + str(colors[i][2]) + ") !important ;\n"
+            opacity = "    fill-opacity: 0.75;\n"
+            end = "}\n"
+            f.write(start + fill + opacity + end)
+
+# Creates the HTML-file from start and end text files in the source directory
+# Includes an option for each volume layer combination from the data
+def createHTML(VID, LID):
+    with open("../src/htmlstart.txt", 'r') as file1:
```

Comment:
with such a design, you enforce that the script is executed in that exact directory, a cleaner way to do this would be to define a `resources` directory path from which it is taken.